src/app/components/Embeds/EmbedHtml/index.stories.tsx

```diff
@@ -21,6 +21,9 @@ export const IDT2QuotePic = () => (
   <EmbedHtml embeddableContent={quotePicEmbedHtml} />
 );
 
-export const Riddle = () => <EmbedHtml embeddableContent={riddleHTML} />;
+export const Riddle = {
+  render: () => <EmbedHtml embeddableContent={riddleHTML} />,
+  parameters: { chromatic: { disableSnapshot: true } },
```

Comment:
We have   `parameters: { chromatic: { disable: true } }` above on line 13 which I think applies to all these story files. Do we want to remove this from line 13? I'm not sure the difference between this and what you've added here for the Riddle component.

Reply:
> We have `parameters: { chromatic: { disable: true } }` above on line 13 which I think applies to all these story files. Do we want to remove this from line 13? I'm not sure the difference between this and what you've added here for the Riddle component.

I think this setting doesn't actually work, I've removed it

Reply:
Hmm it did work, I wonder why the riddle one was showing up then